docker-compose.yml

```diff
@@ -23,6 +23,8 @@ services:
       DEFAULT_LOCALE: ${DOCKER_COMPOSE_APP_DEFAULT_LOCALE:-en}
       SECRET_KEY_BASE: ${DOCKER_COMPOSE_APP_SECRET_KEY_BASE:-NDliN2UzNjAzMTlmMWIxODZhNzRjMDIzMDY2OTQ5N2Q4NGU3MjUxZjM0YmI0ZGY1}
       URL_PORT: ${DOCKER_COMPOSE_APP_URL_PORT:-4000}
+      NLP_WORD_TAGGER_MAX_INPUT: 600
+      VITE_NLP_WORD_TAGGER_MAX_INPUT: 600
```

Comment:
VITE needs this prefix. Either we use the VITE variable also inside the server/backend or we need these two.

Reply:
You could set on to the other in the docker-compose.yml. But I'm also fine with keeping it like this.

Reply:
I think I would keep it like this, as we also need two variables for prod: 1) during the build (Vite for GitHub) and 2) during runtime for the server (elixir)